cli/cmd/alert_rules.go

```diff
@@ -0,0 +1,350 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/api"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// alert-rules command is used to manage lacework alert rules
+	alertRulesCommand = &cobra.Command{
+		Use:     "alert-rule",
+		Aliases: []string{"alert-rules", "ar"},
+		Short:   "manage alert rules",
+		Long:    "Manage routing events to channels using alert rules.",
+	}
+
+	// list command is used to list all lacework alert rules
+	alertRulesListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "list all alert rules",
+		Long:    "List all alert rules configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			alertRules, err := cli.LwApi.V2.AlertRules.List()
+			if err != nil {
+				return errors.Wrap(err, "unable to get alert rules")
+			}
+			if len(alertRules.Data) == 0 {
+				msg := `There are no alert rules configured in your account.
+
+Get started by integrating your alert rules to manage alerting using the command:
+
+    $ lacework alert-rule create
+
+If you prefer to configure alert rules via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Alert Rules.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(alertRules)
+			}
+
+			var rows [][]string
+			for _, rule := range alertRules.Data {
+				rows = append(rows, []string{rule.Guid, rule.Filter.Name, rule.Filter.Description, rule.Filter.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "DESCRIPTION", "ENABLED"}, rows))
```

Comment:
I think the description could be too long for displaying it in the `list` command. Should we NOT display it and let the user run a `show` command for showing more details?
```suggestion
			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, rows))
```

Reply:
Maybe instead, we could show the severities? 🤔  Thoughts?

Reply:
I don't think severities would fit well with the "headers". IMO it should live in "details"